Pipfile

```diff
@@ -16,6 +16,8 @@ colorama = "*"
 verboselogs = "*"
 crcmod = "*"
 tinyaes = "*"
+requests = "*"
```

Comment:
We use `urllib.request` in other places, so if we are including the requests package, we should probably avoid importing `urllib.request` throughout.